ui/store/institutional/institution-background.test.js

```diff
@@ -97,4 +123,52 @@ describe('Institution Actions', () => {
       expect(setWaitForConfirmDeepLinkDialog).toBeDefined();
     });
   });
+
+  describe('#showInteractiveReplacementTokenBanner', () => {
+    it('should test showInteractiveReplacementTokenBanner action', async () => {
+      const dispatch = jest.fn();
+
+      await showInteractiveReplacementTokenBanner({
+        url: 'testUrl',
+        oldRefreshToken: 'testToken',
+      })(dispatch);
+
+      expect(submitRequestToBackground).toHaveBeenCalledWith(
+        'showInteractiveReplacementTokenBanner',
+        [{ url: 'testUrl', oldRefreshToken: 'testToken' }],
+      );
+    });
+  });
+
+  describe('#setTypedMessageInProgress', () => {
```

Comment:
QUESTION: In the code, 
```
export function setTypedMessageInProgress(msgId: string) {
  return async (dispatch: MetaMaskReduxDispatch) => {
    dispatch(showLoadingIndication());
    try {
      await submitRequestToBackground('setTypedMessageInProgress', [msgId]);
    } catch (error: any) {
      log.error(error);
      dispatch(displayWarning(error.message));
    } finally {
      await forceUpdateMetamaskState(dispatch);
      dispatch(hideLoadingIndication());
    }
  };
}
```

do we need to `await forceUpdateMetamaskState(dispatch);` if we have error? i feel like this should be removed under `await submitRequestToBackground('setTypedMessageInProgress', [msgId]);`

wdyt?

Reply:
It's correct to have the `await forceUpdateMetamaskState(dispatch);` in the `finally` block 👍🏼